fiorella/src/styles/layout/_header.scss

```diff
@@ -0,0 +1,294 @@
+    font-size: 15px;
+    text-transform: uppercase;
+  }
+
+  ::placeholder { /* Chrome, Firefox, Opera, Safari 10.1+ */
+    color: $color-black-solid;
+    opacity: 1; /* Firefox */
+  }
+
+  :-ms-input-placeholder { /* Internet Explorer 10-11 */
+    color: $color-black-solid;
+  }
+
+  ::-ms-input-placeholder { /* Microsoft Edge */
+    color: $color-black-solid;
+  }
+}
+
+.mobile-nav {
+  border-bottom-style: solid;
+  border-bottom-color: $color-black-solid;
+  display: flex;
+  flex-direction: column;
+  padding-bottom: 10px;
+  margin-bottom: 10px;
+
+  a {
+    text-transform: uppercase;
+  }
+}
+
+.mobile-user-buttons {
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 20px;
+
+  a {
+    color: $color-featured;
+  }
+}
+
+.mobile-info-buttons {
+  margin-top: 10px;
+}
+
+.mobile-top-section {
+  display: flex;
+  align-items: center;
+  background-color: $color-featured;
+  height: 40px;
+  justify-content: center;
+  position: relative;
+
+  button {
+    @extend %transparent-button;
+  }
+
+  p {
+    background-color: $color-featured;
+    border-style: none;
+    border-radius: 50%;
+    color: $color-white-solid;
+    display: inline;
+    padding: 4px 10px;
+  }
+
+  .header-card-button {
+    @extend %positioning;
+    justify-content: flex-end;
+    padding-right: 10px;
+
+    p {
+      display: inline;
+      font-size: 14px;
+      margin: 0;
+    }
+  }
+
+  img {
+    width: 100%;
+  }
+}
+
+.home-screen-mobile-search {
+  margin: 10px 8px 0;
+}
+
+.menu-open {
+  @extend %positioning;
+  justify-content: flex-start;
+  z-index: 1;
+}
+
+@media #{$small-desktop-view} {
+
+  .show-menu {
+    width: 0;
+    padding: 0;
+  }
+
+  .mobile-top-section {
+    display: none;
+  }
+
+  .header-container {
+    display: flex;
+    align-items: center;
+    background-color: rgba(255, 255, 255, 0.5);
+    border-bottom-style: solid;
+    border-bottom-color: $color-featured;
+    font-family: $font-header;
+
+    @include reset-ul(inline);
+  }
+
+  .search-container {
+    flex: 1;
+    border-bottom-style: none;
+
+    form {
+      display: inline;
+    }
+
+    .desktop-btn-search {
+      display: inline;
+    }
+
+    .mobile-btn-search {
+      display: none;
+    }
+  }
+
+  .header-buttons-container {
+    display: flex;
+    flex: 1;
+    justify-content: flex-end;
+
+    a {
+      color: $color-black-solid;
+      margin-right: 20px;
+      text-decoration: none;
+    }
+  }
+
+  .header-card-button {
+    display: inline;
+    margin-left: 10px;
+
+    p {
+      background-color: $color-featured;
+      border-style: none;
+      border-radius: 50%;
+      color: $color-white-solid;
+      display: inline;
+      padding: 4px 10px;
+    }
+  }
+
+  .homepage-logo {
+    display: flex;
+    justify-content: center;
+    height: 80px;
+    padding: 10px;
+    width: 100%;
+    margin: 20px 0;
+
+    img {
+      height: 100%;
+    }
+  }
+
+  .navbar {
+    display: flex;
+    justify-content: center;
```

Comment:
Properties should be ordered display, font-family, justify-content, text-transform